### PR TITLE
[no ticket][risk=no] Add another cdr version in perf

### PR DESCRIPTION
**Pull request by aweng98**

### CODE REVIEW

File: api/config/cdr_config_perf.json [hunk 1]
```diff
@@ -69,6 +69,20 @@
       "cdrDbName": "synth_r_2019q4_14",
       "hasFitbitData": true,
       "hasCopeSurveyData": true
+    },
+    {
+      "cdrVersionId": 5,
+      "isDefault": true,
```
Comment (jmthibault79) [inline api/config/cdr_config_perf.json:75]: We can have only one "default" version per tier, so this needs to be false.

**Review by jmthibault79 — APPROVED** [review]
Once you make that change, run this from the api directory to apply the changes to Perf:
```
gcloud auth login youruser@pmi-ops.org
docker-sync stop; docker-sync clean; docker-sync start
./project.rb update-cdr-config --project all-of-us-rw-perf
```

If this passes (no errors and you see a message like "added CDR Version") then you can merge.

File: api/config/cdr_config_perf.json (hunk 1 above)
Comment (aweng98) [reply api/config/cdr_config_perf.json:74]: 👍🏻  I made v4 as default. removed default from v3.

**Review by aweng98 — COMMENTED** [review]
`update-cdr-config` ran successfully.

File: api/config/cdr_config_perf.json (hunk 1 above)
Comment (jmthibault79) [reply api/config/cdr_config_perf.json:74]: that works

**Pull request merged**